lib/jbuild

```diff
@@ -0,0 +1,31 @@
+(* -*- tuareg -*- *)
+#require "unix"
+
+let flags pkgs =
+  let cmd = "ocamlfind ocamlc -verbose" ^ (
+    List.fold_left (fun acc pkg -> acc ^ " -package " ^ pkg) "" pkgs
+  ) in
+  let ic = Unix.open_process_in
+    (cmd ^ " | grep -oEe '-ppx \"[^\"]+\"'")
```

Comment:
As we already discussed, the use of `grep` here will make these packages not windows compatible.
However this is only a temporary thing and it is only in the old package.

One possible failure here is if the result of the ocamlfind blablabla thing contains `-ppx "escaped string with escaped quote"`. But, again, I think it is a sufferable pain for a transitory thing